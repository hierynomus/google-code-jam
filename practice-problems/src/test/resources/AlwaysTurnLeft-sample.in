2
WRWWLWWLWWLWLWRRWRWWWRWWRWLW WWRRWLWLWWLWWLWWRWWRWWLW
WW WW