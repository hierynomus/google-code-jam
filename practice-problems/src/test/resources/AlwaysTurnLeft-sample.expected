Case #1: 
ac5
386
9c7
e43
9c5
Case #2: 
3
